server/services/prisonService.ts

```diff
@@ -261,6 +262,9 @@ export default class PrisonService {
 
   getEducations(prisonerNumber: string | string[], user: ServiceUser): Promise<Education[]> {
     const prisonerNumbers = [prisonerNumber].flat()
-    return this.prisonApiClient.getEducations(prisonerNumbers, user)
+    return this.prisonApiClient
+      .getEducations(prisonerNumbers, user)
+      .then(edu => edu.filter(e => e.studyArea && e.educationLevel && isBefore(parseDate(e.endDate), new Date())))
+      .then(edu => _.uniqBy(edu, e => e.bookingId + e.studyArea + e.educationLevel))
```

Comment:
Could there ever be a reason to fetch prisoner education without this filter? Just wondering if this logic is best placed in the service?